Gems/PhysX/Code/Source/Joint/Configuration/PhysXJointConfiguration.cpp

```diff
@@ -13,12 +13,12 @@
 
 namespace PhysX
 {
-    JointGenericProperties::JointGenericProperties(GenericJointFlag flags, float forceMax, float torqueMax)
+    JointGenericProperties::JointGenericProperties(GenericJointFlag flags, float forceMax, float torqueMax, AZStd::string joint_name)
         : m_flags(flags)
         , m_forceMax(forceMax)
         , m_torqueMax(torqueMax)
     {
-
+        m_jointname = joint_name;
```

Comment:
The initialization of `m_jointName` could be done like the other members, and since the string parameter will contain a copy (since it's not passed as reference) it can move the string to avoid another copy.

````
    JointGenericProperties::JointGenericProperties(GenericJointFlag flags, float forceMax, float torqueMax, AZStd::string jointName)
        : m_flags(flags)
        , m_forceMax(forceMax)
        , m_torqueMax(torqueMax)
        , m_jointName(AZStd::move(jointName))
    {
    }
````